src/main/java/io/citrine/jcc/search/pif/result/PifMultiSearchResultElement.java

```diff
@@ -0,0 +1,23 @@
+package io.citrine.jcc.search.pif.result;
+
+import io.citrine.jcc.search.core.result.BaseMultiSearchResultElement;
+
+/**
+ * Class to store the information in a single search result as a part of a PIF multi-query.
+ *
+ * @author Kyle Michel
+ */
+public class PifMultiSearchResultElement extends BaseMultiSearchResultElement<PifSearchResult> {
```

Comment:
So that we can do

```
PifMultiSearchResult searchResult = Query.execute();
for (PifMultiSearchResultElement i : searchResult) { // Work }
```

Rather than

```
PifMultiSearchResult searchResult = Query.execute();
for (BaseMultiSearchResultElement<PifSearchResult> i : searchResult) { // Work }
```